test/ppspp_options_SUITE.erl

```diff
@@ -56,7 +57,7 @@ getters(_Config) ->
     Swarm_id = convert:hex_string_to_padded_binary(Root_Hash),
     Options = ppspp_options:use_default_options(Swarm_id),
     %% confirm that we can unwrap options from within a Datagram-style orddict
-    Options = ppspp_options:get_options([{foo, bar}, {fu, baa}, Options]),
+    Options = ppspp_options:get_options(#{foo => bar, fu => baa, options => Options}),
     % check returned values against header file defaults
     ?PPSPP_DEFAULT_CHUNK_METHOD = ppspp_options:get_chunk_addressing_method(Options),
```

Comment:
Line 62 is too long:     ?PPSPP_DEFAULT_CHUNK_METHOD = ppspp_options:get_chunk_addressing_method(Options),.